### PR TITLE
Added error message when quickbuying a missing id

### PR DESCRIPTION
**Pull request by VirtualSatai**

### CODE REVIEW

**Comment by VirtualSatai** [comment]
This fixes #120

File: stregsystem/templates/stregsystem/error_productdoesntexist.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <div>This incident will be reported.</div>
+{% endblock %}
+{% block message %}
+    <div>
+        <center>
+            <b>Du forsøgte at købe id: {{ failedProduct }}, men dette product blev ikke fundet ¯\_(ツ)_/¯.</b>
+        </center>
+    </div>
+    <br />
+    Hvis du mener at <i>det er for galt</i>, anbefaler F-Klubbens IT, at du venter, indtil nogen g&oslash;r noget ved problemet.
```
Comment (hrjakobsen) [inline stregsystem/templates/stregsystem/error_productdoesntexist.html]: Why is 'ø' written as '&oslash;' while 'æ' is written 'æ'?

Comment (hrjakobsen) [reply stregsystem/templates/stregsystem/error_productdoesntexist.html]: Argh! Github changed my message. I meant: why is 'ø' written as ```&oslash;```

Comment (josmad) [reply stregsystem/templates/stregsystem/error_productdoesntexist.html]: Half new, half copy-pasta. We should probably align this in all templates.

Comment (VirtualSatai) [reply stregsystem/templates/stregsystem/error_productdoesntexist.html]: Yup, Jonas got it right :p

**Pull request merged**